scrapy/extensions/checksettings.py

```diff
@@ -0,0 +1,54 @@
+"""
+Check settings extension
+
+See documentation in docs/topics/extensions.rst
+"""
+import logging
+import pprint
+import operator
+
+from fuzzywuzzy.fuzz import ratio
```

Comment:
I wonder if it would not be better to just indicate that a setting is unused, and not suggest possible matches, to avoid adding a new dependency.

Reply:
Changed to difflib.